app/pb_kits/playbook/pb_date_range_inline/_date_range_inline.scss

```
@@ -1,3 +1,34 @@
-.pb_date_range_inline {
-
+[class^=pb_date_range_inline_kit] {
+  &[class*=_center]  {
+    & > [class^=pb_caption],
+    & > [class^=pb_body] {
+      text-align: center;
+    }
+    & > [class*=pb_date_range_inline_wrapper] {
+      justify-content: center;
+    }
+  }
+  &[class*=_right] {
+    & > [class^=pb_caption],
+    & > [class^=pb_body] {
+      text-align: right;
+    }
+    & > [class*=pb_date_range_inline_wrapper] {
+      justify-content: flex-end;
+    }
+  }
+  [class^=pb_date_range_inline_wrapper] {
+    display: flex;
+    align-items: center;
+    [class*=pb_date_range_inline_arrow] {
+      margin-left: 4px;
+      margin-right: 4px;
+    }
+    [class*=pb_date_range_inline_timezone] {
+      margin-left: 4px;
+    }
+    [class*=pb_date_range_inline_icon] {
+      margin-right: 4px;
```

I'd like to use variables as much as possible. I'm not sure what 8px would look like...but assuming that looks bad we could at divide `$space_xs` to get 4px.

```suggestion
      margin-left: $space_xs/2;
      margin-right: $space_xs/2;
    }
    [class*=pb_date_range_inline_timezone] {
      margin-left: $space_xs/2;
    }
    [class*=pb_date_range_inline_icon] {
      margin-right: $space_xs/2;
```

---

@jasoncypret That definitely makes sense. Do you think it'd be a bad idea to add a new variable like `$space_xxs` for 4px? I see `$space_xs/2` used in a couple other kits which could be changed to use that variable if it sounds like a good idea.

---

I went ahead and made this change.